tests/test_manifest.py

```diff
@@ -71,7 +69,9 @@ def _build_sdist_and_get_members(self):
 		# Arguments need to build
 		build_arguments = [
 			str("{} -m coverage run").format(sys.executable),
```

Comment:
This could also be improved as per #134 with F-strings.

```suggestion
			f"{str(sys.executable)} -m coverage run",
```